pkg/cloud-init/cloud-init.go

```diff
@@ -572,3 +572,7 @@ func GenerateLocalData(vmiName string, namespace string, data *CloudInitData) er
 	log.Log.V(2).Infof("generated nocloud iso file %s", iso)
 	return nil
 }
+
+func PrepareLocalPath(vmiName string, namespace string) error {
```

Comment:
> This is the reciprocal of removeLocalData, right? Would you place them next to each other to help the reader?

Sure. Initially I did not notice that there was `removeLocalData` function. Will move `PrepareLocalPath` then.

> I don't pretend to understand the entire flow, but I'd expect that if LibvirtDomainManager is now responsible for calling PrepareLocalPath it would also be the entity responsible for cleaning it up?

In fact currently there is no cleanup.  Once cloud-init ISO is generated it stays there since it is used for the whole lifetime of the VM.

Reply:
oh.. `removeLocalData` is used only in the tests. I wish we had had a proper separation between production package and test package :-/